Parse/PFSubclassing.h

```diff
@@ -23,6 +23,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @protocol PFSubclassing
 
+@required
```

Comment:
Technically a protocol is `@required` by default, so it's not 100% necessary. Its a style thing though, I'll let you decided whether or not to keep it.